src/containers/App/App.js

```diff
@@ -57,32 +65,24 @@ class App extends Component {
     });
   };
 
-  /*
-  * This method takes an archive and
-  * returns all of their posts. 
-  * It does not setState.
-  */
-  getAllPosts = async archive => {
-    const posts = await archive.readdir('/posts');
+  refreshTheirPosts = async archive => {
+    await queryFollowers();
+    const posts = await archive.readdir('/theirposts');
     if (posts.length === 0) {
       this.setState({
-        posts: []
+        theirPosts: []
       });
     } else {
       const promises = posts.map(async post => {
-        const postResponse = await archive.readFile(`/posts/${post}`);
+        const postResponse = await archive.readFile(`/theirposts/${post}`);
         return JSON.parse(postResponse);
       });
       const results = await Promise.all(promises);
       return results;
     }
   };
 
-  /*
-  * Exact same as the above function, but
-  * also sets state.
-  */
-  getAllPostsSaved = async archive => {
```

Comment:
This is a new function. Similar to the old refreshPosts, but it also does setState